docs/diagrams/join.md

```diff
@@ -0,0 +1,41 @@
+title etcdadm join https://member1:2379
+
+participant user
+participant remote repo
+participant etcdadm
+participant systemd
+database member1
+database member2
+
+user->user: copy CA cert/key to host
+note over user: all client and peer certificates must be signed by the same CA
+user->etcdadm: join https://member1:2379
+
+group add new member
+etcdadm->member1: add member
+end
+
+group download (if not in cache)
+etcdadm->remote repo: download etcd, etcdctl into local cache
+end
+
+group install artifacts
+etcdadm->etcdadm: copy etcd, etcdctl from local cache to install directory
+end
+
+group create certificates
+etcdadm->etcdadm: create CA certificate
```

Comment:
Join doesn't create a CA cert, right? It uses the CA cert passed in by the user, the one that was created in init?

Reply:
Right. Join doesn't create a CA cert. Thanks for catching this!